fitsio/__init__.py

```diff
@@ -4,7 +4,7 @@
 usage.
 """
 
-__version__='0.9.12'
+__version__='1.0.0rc1'
```

Comment:
note there is already a tag `v1.0.0rc1`, maybe bump to `rc2`?

Reply:
Ack yes.